runtime/gc_glue_java/EnvironmentDelegate.cpp

```diff
@@ -111,7 +110,6 @@ MM_EnvironmentDelegate::attachVMThread(OMR_VM *omrVM, const char  *threadName, u
 	if (JNI_OK != javaVM->internalVMFunctions->attachSystemDaemonThread(javaVM, &vmThread, threadName)) {
 		return NULL;
 	}
-	javaVM->internalVMFunctions->internalReleaseVMAccessInJNI(vmThread);
```

Comment:
Doesn't removing this change the program meaning?  ie: for the non `J9VM_INTERP_ATOMIC_FREE_JNI` case

Reply:
The old API used to call `internalReleaseVMAccessNoMutex` if the thread had vm access.

Reply:
In older releases, there was no line there at all - I had added this for safety (releaseVMAccessInJNI is essentially "release VM access if I have it and mark me as in the VM").

attachSystemDaemonThread guarantees this now, as a consequence of calling initializeAttachedThread at the end of the function.